pyproject.toml

```diff
@@ -32,15 +32,15 @@ https-loader = ["requests"]
 https-loader = "_delb.plugins.https_loader"
 
 [tool.poetry.dependencies]
-python = "~=3.6"
+python = ">=3.6.2,<4.0"
```

Comment:
i don't know your motivation, but Guido once said that a 4.0 will not be incompatible with the previous series. that may be not true in the future, but i take Guido's word as guideline 'til then.

Reply:
I literally just did exactly what poetry told me to do, so I really can't be blamed for starting a beef with Guido!
![Screenshot from 2021-11-11 20-51-48](https://user-images.githubusercontent.com/1922142/141360286-c78b45a4-1225-4e45-b6dd-b1f5d44b36fa.png)

Reply:
```suggestion
python = "^3.6.2"
```
would work, but I think that's [equivalent](https://python-poetry.org/docs/dependency-specification/#caret-requirements) to what poetry told me to write.

Reply:
yes it is.

Reply:
i don't get what poetry is telling there. black defines `python_requires=">=3.6.2"`. and "the current project" (delb i guess) doesn't define what it tells.

Reply:
is it just nitpicky about the micro version that black wants?

Reply:
beats me!